### PR TITLE
Added validation to prevent making group inactive if the group has active roles

### PR DESCRIPTION
**Pull request by danieljames-dj**
This PR is to add validation to prevent making a group inactive if the group has at least one active role.

### CODE REVIEW

File: app/webpacker/components/Panel/pages/RegionManager/index.jsx [hunk 1]
```diff
@@ -22,7 +22,9 @@ const defaultRegion = {
   friendlyId: '',
 };
 
-function UserGroupVisibility({ userGroup, save, sync }) {
+function UserGroupVisibility({
+  userGroup, save, sync, setSaveError,
```
Comment (gregorbg) [inline app/webpacker/components/Panel/pages/RegionManager/index.jsx:26]: I don't see any code change where components pass `setSaveError`...
Where are you passing any value along?

Comment (danieljames-dj) [reply app/webpacker/components/Panel/pages/RegionManager/index.jsx:26]: It's passed in line 134 and 169, but I guess you already saw that, did you mean something else?

Comment (gregorbg) [reply app/webpacker/components/Panel/pages/RegionManager/index.jsx:26]: Yeah but where is that `setSaveError` that is being passed in those lines actually defined?
Was it already existing before?

Comment (danieljames-dj) [reply app/webpacker/components/Panel/pages/RegionManager/index.jsx:26]: Yes, it was already existing earlier. But it was not passing the error to the `Errored` component. I've done it now. Still it's not displaying a proper message, but I think that is fine.

**Review by gregorbg — APPROVED** [review]

**Pull request merged**